src/h5web/vis-packs/core/utils.ts

```diff
@@ -50,16 +50,7 @@ export function createAxisScale(
 ): AxisScale {
   const { type } = config;
 
-  switch (type) {
-    case ScaleType.Linear:
-      return scaleLinear<number>(config);
-    case ScaleType.Log:
-      return scaleLog<number>(config);
-    case ScaleType.SymLog:
-      return scaleSymlog<number>(config);
-    default:
-      throw new Error('Unknown type');
-  }
+  return H5WEB_SCALES[type].createScale(config);
```

Comment:
I kept `createAxisScale` to reduce the diff but we could remove it in favour of a direct call to `H5WEB_SCALES[type].createScale(config)`

Reply:
Hmm, I still like the abstraction, personally. Unless this is now called in very few places?

If we keep it, the problem is more the name... it's been wrong for a while now. Perhaps `createVisScale`? Let's change this in another PR, though.